packages/rum-core/src/common/config-service.js

```diff
@@ -260,6 +260,21 @@ class Config {
 
     return errors
   }
+
+  getLocalConfig() {
+    let config = sessionStorage.getItem(LOCAL_CONFIG_KEY)
+    if (config) {
+      return JSON.parse(config)
+    }
+  }
+
+  setLocalConfig(config) {
+    if (config) {
+      sessionStorage.setItem(LOCAL_CONFIG_KEY, JSON.stringify(config))
+    } else {
+      sessionStorage.removeItem(LOCAL_CONFIG_KEY)
```

Comment:
just worried about whether this might hit us hard at somepoint. Will there be a case when server returns a falsy value and we end up deleting the old config value? If this is used only for tests, lets remove this.

Reply:
Decided to ignore falsy values instead of removing the local config!